evap/grades/tests.py

```diff
@@ -132,6 +132,23 @@ def test_upload_final_grades(self):
         evaluation.publish()
         evaluation.save()
         self.helper_check_final_grade_upload(course, 0)
+    
+    def test_grades_headlines(self):
+        # check midterm headline
+        self.assertEqual(self.course.midterm_grade_documents.count(), 0)
+        semester = baker.make(Semester, pk=1, grade_documents_are_deleted=False)
+        baker.make(Evaluation, course=baker.make(Course, pk=1, semester=semester), state=Evaluation.State.PREPARED)
```

Comment:
This can be shortened to
```python
baker.make(Evaluation, course__pk=1, course__semester__pk=1, state=...)
```
Additionally, the `grade_documents_are_deleted` is `False` by default